plugin.py

```diff
@@ -1,14 +1,61 @@
 import shutil
+import os
 import sublime
 from LSP.plugin.core.handlers import LanguageHandler
 from LSP.plugin.core.settings import ClientConfig, LanguageConfig
 
 config_name = 'metals-sublime'
 server_name = 'metals-sublime'
 service_name = "metals-sublime"
+coursierPath = os.path.join(os.path.dirname(__file__), './coursier')
+service_path = []
+
+def metals_is_running() -> bool:
+    return shutil.which(service_name) is not None
+
+def get_java_path(java_path: 'Optional[str]') -> 'Optional[str]':
+    if java_path:
+        return java_path + "/bin/java"
+    elif shutil.which("java") is not None:
+        return "java"
+    else:
+        return None
+
+def plugin_loaded():
+    global service_path
+    plugin_settings = sublime.load_settings("metals-sublime.sublime-settings")
+
+    java_path = get_java_path(plugin_settings.get('java_home'))
+    if not java_path :
+        window.status_message("Please install java or set the 'java_home' setting")
+
+    server_version = plugin_settings.get('server_version')
+    if not server_version :
+        window.status_message("'server_version' should be set")
+
+    server_properties = prepare_server_properties(plugin_settings.get('server_properties', []))
+
+    service_path = [
+        java_path,
+        "-jar",
+        coursierPath,
+        "fetch",
+        "-p",
+        "--ttl",
+        "Inf",
+        "org.scalameta:metals_2.12:{}".format(server_version),
+        "-r",
+        "bintray:scalacenter/releases",
+        "-r",
+        "sonatype:public",
+        "-r",
+        "sonatype:snapshots",
+        "-p"
+    ]
+
 metals_config = ClientConfig(
     name=config_name,
-    binary_args=[service_name],
+    binary_args=service_path,
```

Comment:
The VS Code extension launches `java` twice:
```
metals_classpath=$(java -jar coursier fetch org.scalameta:metals...)
java -classpath $metals_classpath scala.meta.metals.Main
```
The command that Sublime LSP should invoke is the second command with the Metals Main.